{"name": "searching for invalid product", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=113.0.5672.127)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00B58893+48451]\n\t(No symbol) [0x00AEB8A1]\n\t(No symbol) [0x009F5058]\n\t(No symbol) [0x009E7BB6]\n\t(No symbol) [0x009E78D9]\n\t(No symbol) [0x009F6460]\n\t(No symbol) [0x00A4C038]\n\t(No symbol) [0x00A3A0B6]\n\t(No symbol) [0x00A17E08]\n\t(No symbol) [0x00A18F2D]\n\tGetHandleVerifier [0x00DB8E3A+2540266]\n\tGetHandleVerifier [0x00DF8959+2801161]\n\tGetHandleVerifier [0x00DF295C+2776588]\n\tGetHandleVerifier [0x00BE2280+612144]\n\t(No symbol) [0x00AF4F6C]\n\t(No symbol) [0x00AF11D8]\n\t(No symbol) [0x00AF12BB]\n\t(No symbol) [0x00AE4857]\n\tBaseThreadInitThunk [0x770B7D59+25]\n\tRtlInitializeExceptionChain [0x7720B74B+107]\n\tRtlClearBits [0x7720B6CF+191]\n\n", "trace": "  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features\\environment.py\", line 25, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(),name='failed_screensot.png',attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 927, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 938, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given i am in home page", "status": "passed", "start": 1684913713804, "stop": 1684913713815}, {"name": "When entering invalid product say \"hp\" in search box field", "status": "passed", "start": 1684913713815, "stop": 1684913713924}, {"name": "And clicking on search icon", "status": "passed", "start": 1684913713924, "stop": 1684913715018}, {"name": "Then proper error message has to display", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=113.0.5672.127)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00B58893+48451]\n\t(No symbol) [0x00AEB8A1]\n\t(No symbol) [0x009F5058]\n\t(No symbol) [0x009E7BB6]\n\t(No symbol) [0x009E78D9]\n\t(No symbol) [0x009F6460]\n\t(No symbol) [0x00A4C038]\n\t(No symbol) [0x00A3A0B6]\n\t(No symbol) [0x00A17E08]\n\t(No symbol) [0x00A18F2D]\n\tGetHandleVerifier [0x00DB8E3A+2540266]\n\tGetHandleVerifier [0x00DF8959+2801161]\n\tGetHandleVerifier [0x00DF295C+2776588]\n\tGetHandleVerifier [0x00BE2280+612144]\n\t(No symbol) [0x00AF4F6C]\n\t(No symbol) [0x00AF11D8]\n\t(No symbol) [0x00AF12BB]\n\t(No symbol) [0x00AE4857]\n\tBaseThreadInitThunk [0x770B7D59+25]\n\tRtlInitializeExceptionChain [0x7720B74B+107]\n\tRtlClearBits [0x7720B6CF+191]\n\n", "trace": "  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features\\environment.py\", line 25, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(),name='failed_screensot.png',attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 927, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 938, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\praka\\Desktop\\qafox\\BDD_framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1684913715019, "stop": 1684913846824}], "start": 1684913711319, "stop": 1684913850305, "uuid": "8f265879-9237-40b6-8bcc-55b4d03cca71", "historyId": "5da8af0d4ba9aedb2ac1bb861a1d5964", "fullName": "search feature: searching for invalid product", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "search"}, {"name": "feature", "value": "search feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}